### DISCUSSION TITLE
Condensed view/list of aliases on the web page

### DISCUSSION THREAD

**Discussion by ghost** [simple-login/app#946] [Feature Requests]
First off, thank you for SimpleLogin!

It would be great to have an option to view all the aliases in more compact or condensed form. For example, each alias would have on one line across the page: alias, first several characters of alias description, enabled/disabled flag, a "details" link to show more details about the alias.

**Comment by ap-andersson** [comment]
I am currently using another similar service and this is one of the few things that jumped out at me while trying SImpleLogin and that I found a bit irritating. I do not get a good overview of my aliases on the page. There should probably be a toggle for the view where one is more of a table-like format and one is the current one.

**Comment by kyleinprogress** [comment]
Definitely agree with this. Having the ability to see a list / table of my aliases and manage them from there would be super helpful.

**Comment by c1chy** [comment]
I have exactly the same experience. SL is packed with features and planning to have dozens of aliases I need a compact view to easily see them (not zigzagging with my eyes).

**Comment by bmorgy** [comment]
Wholly agree.  It was my first thought when looking at the service: _why aren't the alias listed in a table format_?   Also, I find the "email sent to alias" line directly under the alias name misleading.  I'm more interested in what mailbox is associated with this alias, and finding which mailboxes are using which aliases.  This could be easily resolved with a table, especially if it allows sorting, filtering, etc. on the columns.

**Comment by cautious-bread** [comment]
Adding my voice to this as I just found the older thread over at https://github.com/simple-login/app/discussions/520 and also created a post on the forums.
Table view needs to be a thing.

**Comment by Blizzke** [comment]
Adding my vote here too. I am very happy about SimpleLogin (and I ditched another product that shall not be named to get here), but my aliases are getting a bit out of hand at the moment. Please provide a table view.

**Comment by CicerBro** [comment]
+1. I like the Addy.io list view a lot more than the current SL one.

**Comment by valZho** [comment]
Migrating from a simple forwarding solution on my custom domain with over 500 forwarders (and multiple domains). The card view is borderline unmaintainable (it would be completely unmaintainable if not for the search).

**Comment by pwilliamson19** [comment]
I agree - the present view is not helpful if you have lots of aliases. A few sort options might be useful, and options to select the fields displayed. I definitely support this request.

**Comment by jv75** [comment]
I also would like to request a more condensed table view. This topic was started two and a half year ago, why is this so difficult to implement ?
